src/d3d9/d3d9_device.cpp

```diff
@@ -4252,8 +4239,11 @@ namespace dxvk {
          shouldFlush &= !pResource->GetReadOnlyLocked(Subresource);
          shouldFlush &= !pResource->IsManaged() || m_d3d9Options.evictManagedOnUnlock;
 
-    if (shouldFlush)
+    if (shouldFlush) {
         this->FlushImage(pResource, Subresource);
+        if (pResource->IsManaged())
```

Comment:
!= DEFAULT

Reply:
The dirty box is also used for UpdateTexture and UpdateSurface so I thought we cant clear that but that line isnt called for backed system memory textures anyway so your suggestion is fine too.

Reply:
Oh right, this is for the funny evictManagedOnUnlock path. Ignore this then.